cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -490,6 +490,11 @@ private class CampaignManagerImpl(
 
         // handle gdpr
         response.gdpr?.apply {
+            gdprConsentStatus?.let { gdprCS ->
+                val updatedGdprConsentStatus = gdprCS.copy(applies = applies)
```

Comment:
@bohdan-go-wombat I suggest renaming`CS` to `ConsentStatus`

Reply:
@Nevazhnovu in that case we will have shadowed naming of `gdprConsentStatus` from the `CampaignManager` itself, which acceptable, but not really good from the readability point of view, since you have 2 variables with the same name

although, they both mean the same thing (only that the one inside curly brackets will just be non nullable), I just think that `CS` abbreviation is pretty obvious in this case